modules/MinimumSpanningTree/pom.xml

```diff
@@ -0,0 +1,118 @@
+    <dependency>
+      <groupId>org.gephi</groupId>
+      <artifactId>graph-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.netbeans.api</groupId>
+      <artifactId>org-openide-util-lookup</artifactId>
+    </dependency>
+    <dependency>
+        <groupId>org.gephi</groupId>
+        <artifactId>io-importer-api</artifactId>
+    </dependency>
+    <dependency>
+        <groupId>org.gephi</groupId>
+        <artifactId>io-importer-plugin</artifactId>
+    </dependency>
+    <dependency>
+        <groupId>org.gephi</groupId>
+        <artifactId>io-processor-plugin</artifactId>
+    </dependency>
+    <dependency>
+        <groupId>org.netbeans.modules</groupId>
+        <artifactId>org-netbeans-modules-masterfs</artifactId> <!-- See http://wiki.netbeans.org/UsingFileSystemsMasterfs -->
+        <scope>test</scope>
+    </dependency>
+    <dependency>
+        <groupId>junit</groupId>
+        <artifactId>junit</artifactId>
+        <version>4.10</version>
+        <scope>test</scope>
+    </dependency>
+        <dependency>
+            <groupId>org.gephi</groupId>
+            <artifactId>filters-api</artifactId>
+            <scope>test</scope>
+            <type>jar</type>
+        </dependency>
+        <dependency>
+            <groupId>org.gephi</groupId>
+            <artifactId>filters-plugin</artifactId>
+            <scope>test</scope>
+            <type>jar</type>
+        </dependency>
+        <dependency>
+            <groupId>org.gephi</groupId>
+            <artifactId>filters-impl</artifactId>
+            <scope>test</scope>
+            <type>jar</type>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>nbm-maven-plugin</artifactId>
+                <configuration>
+                    <licenseName>gpl-3.0</licenseName>
+                    <licenseFile>gpl-3.0.txt</licenseFile>
+                    <author>Carl Schroedl carlschroedl@gmail.com</author>
+                    <sourceCodeUrl>https://github.com/carlschroedl/gephi-plugins</sourceCodeUrl>
+                    <homePageUrl>https://github.com/carlschroedl/gephi-plugins/tree/minimum-spanning-tree-plugin</homePageUrl>
+                    <publicPackages>
+                        <publicPackage>com.carlschroedl.gephi.plugin.minimumspanningtree</publicPackage>
+                    </publicPackages>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.15</version>
+            </plugin>
+        </plugins>
+    </build>
+    <reporting>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>cobertura-maven-plugin</artifactId>
+                <version>2.6</version>
```

Comment:
Unfortunately, when triggering code coverage collection in the NetBeans UI, it disgregards the version of the coverage plugin that is specified in the pom. Instead, NetBeans uses the most recent stable release of the plugin. In this case, there are integration problems between NetBeans 8.2 and the most recent stable release of the cobertura-maven-plugin (2.8). By running the `cover.sh` script I included above, a user can at least view their code lines painted in red, yellow, or green depending on test coverage.